package.json

```diff
@@ -28,9 +28,11 @@
     "seed": "ts-node prisma/seed.ts"
   },
   "dependencies": {
-    "@chakra-ui/react": "^1.7.4",
+    "@chakra-ui/icons": "^1.1.1",
```

Comment:
Do we need the chakra icons for anything? If not, I think we should be okay to remove them.

Reply:
I've been using them in a few places